src/main/kotlin/org/flaxo/plagiarism/main.kt

```diff
@@ -139,6 +139,19 @@ private fun showGraphVisualizationTool(graphUrl: String?) {
                 +"enabled"
             }
         }
+        div(classes = "input-group") {
+            div {
+                +"Hide extra nodes"
```

Comment:
Maybe inverted `Show active` checkbox will be a little bit clearer?

Reply:
So you suggest to make the `Show active` checkbox with the false default value?

Reply:
I changed it to `Show active` and `true`